### PR TITLE
DHCP fingerprints for Polycom SoundStation IP

### PR DESCRIPTION
**Pull request by dmoinescu-r7**
## Description
DHCP fingerprints for Polycom SoundStation IP, as observed in DHCP network traffic.

## How Has This Been Tested?
Ran `recog_verify`, `recog_standardize` and `update_cpes.py`

## Types of changes

- New feature (non-breaking change which adds functionality)

## Checklist:

- [x] I have updated the documentation accordingly (or changes are not required).
- [x] I have added tests to cover my changes (or new tests are not required).
- [x] All new and existing tests passed.

### CODE REVIEW

File: xml/dhcp_vendor_class.xml [hunk 1]
```diff
@@ -144,6 +144,17 @@
...
 
+  <fingerprint pattern="^Polycom-SSIP([4-7]000)$">
+    <description>Polycom SoundStation IP Phone</description>
+    <example hw.product="6000" hw.model="6000">Polycom-SSIP6000</example>
+    <example hw.product="7000" hw.model="7000">Polycom-SSIP7000</example>
+    <param pos="0" name="hw.device" value="VoIP"/>
+    <param pos="0" name="hw.vendor" value="Polycom"/>
+    <param pos="0" name="hw.family" value="SoundStation IP"/>
+    <param pos="1" name="hw.model"/>
+    <param pos="0" name="hw.product" value="{hw.model}"/>
```
Comment (mkienow-r7) [inline xml/dhcp_vendor_class.xml:155]: `hw.product` should be `{hw.family} {hw.model}` in order to match existing CPE values.
```
h:polycom:soundstation_ip_4000
h:polycom:soundstation_ip_5000
h:polycom:soundstation_ip_6000
h:polycom:soundstation_ip_7000
```

**Review by mkienow-r7 — CHANGES_REQUESTED** [review]

File: xml/dhcp_vendor_class.xml (hunk 1 above)
Comment (dmoinescu-r7) [reply xml/dhcp_vendor_class.xml]: Done

**Comment by mkienow-r7** [comment]
The tests are failing since the example `hw.product` were not updated.

```
$ bundle exec ./bin/recog_verify xml/dhcp_vendor_class.xml
xml/dhcp_vendor_class.xml:148: FAIL: 'Polycom SoundStation IP Phone' failed to find expected capture group hw.product '6000'. Result was SoundStation IP 6000
xml/dhcp_vendor_class.xml:148: FAIL: 'Polycom SoundStation IP Phone' failed to find expected capture group hw.product '7000'. Result was SoundStation IP 7000
xml/dhcp_vendor_class.xml: SUMMARY: Test completed with 81 successful, 0 warnings, and 2 failures
```

**Comment by dmoinescu-r7** [comment]
> The tests are failing since the example `hw.product` values were not updated.
Thanks, sorry about that. I'd seen the test failures, was wondering why (I must've forgotten to re-run `recog_verify` after the latest updates). Fixed now

**Review by mkienow-r7 — APPROVED** [review]
Thank you for the contribution @dmoinescu-r7!

**Pull request merged**